docs/02-hugo.Rmd

```diff
@@ -1,28 +1,89 @@
 # Hugo
 
-What is a static site generator?
+In this chapter, we will introduce [Hugo](https://gohugo.io), the static site generator on which `blogdown` is based. 
 
-What is Hugo?
+"Static site generator" refers to the way that a website's content is generated. On dynamically generated sites, the content is rendered every time a user visits the site. Alternatively, on statically generated sites, the content is only rendered when the content changes, making it ready to access when a user visits the site^[More information here: https://gohugo.io/overview/introduction/]. 
 
-Why Hugo?
+Hugo is one example of a static site generator. It provides the framework for a variety of website types including, but certainly not limited to, blogs.
+
+We're fans of Hugo because it's
```

Comment:
The copyeditor will ask you to expand `We're` to `We are`, and `it's` to `it is` for sure :)

That said, I'd use either a paragraph instead of bullet points to explain the benefits of Hugo, or more detailed bullets. These short bullets were just my quick braindump and they have to be elaborated a bit.